out.csv

```diff
@@ -0,0 +1,51 @@
+Locker,Zone,Status
+1234,A,not-in-use
+12,B,not-in-use
+13,B,not-in-use
+14,B,not-in-use
+15,B,not-in-use
+16,B,not-in-use
+17,B,not-in-use
+18,B,not-in-use
+19,B,not-in-use
+20,B,not-in-use
+21,B,not-in-use
+22,B,not-in-use
+23,B,not-in-use
+24,B,not-in-use
+25,B,not-in-use
+26,B,not-in-use
+27,B,not-in-use
+28,B,not-in-use
+29,B,not-in-use
+30,B,not-in-use
+31,B,not-in-use
+1234,A,not-in-use
+1234,A,not-in-use
+14,B,not-in-use
+15,B,not-in-use
+16,B,not-in-use
+17,B,not-in-use
+18,B,not-in-use
+1234,A,not-in-use
+12,B,not-in-use
+13,B,not-in-use
+14,B,not-in-use
+15,B,not-in-use
+16,B,not-in-use
+17,B,not-in-use
+18,B,not-in-use
+19,B,not-in-use
+20,B,not-in-use
+21,B,not-in-use
+22,B,not-in-use
+23,B,not-in-use
+24,B,not-in-use
+25,B,not-in-use
+26,B,not-in-use
+27,B,not-in-use
+28,B,not-in-use
+29,B,not-in-use
+30,B,not-in-use
+31,B,not-in-use
+2222,A,not-in-use
```

Comment:
I dont think we need this file. This file should be added to .gitignore as we dont need git to keep track of it I guess. What do you think about it?

Reply:
resolved